Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:12
```

Comment:
@sankita15 What is the java version that we agreed on? Is Java 12 ok?

Reply:
Its java 11